api/nakadi-event-bus-api.yaml

```diff
@@ -953,29 +965,27 @@ definitions:
             `EventTypeSchema`.
             Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
 
-          - `data`: Events of this type will be DataChangeEvents. `effective_schema` contains `metadata`, and
-            adds fields `data_op` and `data_type`. The passed EventTypeSchema defines the schema of `data`.
+          - `data`: Events of this category will be DataChangeEvents. `effective_schema` contains `metadata`,
+            and adds fields `data_op` and `data_type`. The passed EventTypeSchema defines the schema of
+            `data`.
             Default validation_strategy for this `EventType` is `[{name: 'datachange-schema-validation'}]`.
 
-          - `business`: Events of this type will be BusinessEvents. `effective_schema` contains `metadata`
-            and any additionally defined properties passed in the `EventTypeSchema` directly on top level of
-            the Event. If name conflicts arise, creation of this EventType will be rejected.
+          - `business`: Events of this category will be BusinessEvents. `effective_schema` contains
+            `metadata` and any additionally defined properties passed in the `EventTypeSchema` directly on
+            top level of the Event. If name conflicts arise, creation of this EventType will be rejected.
             Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
-      topic:
-        type: string
-        description: |
-          References the name of the underlying topic (implementation dependent) that is holding the stream
-          of Events for this EventType.
-
-          Usually is the same as the name property. This field is exposed mostly for monitoring purposes.
       effective_schema:
         type: string
         #read-only: true
         description: |
           The effective schema of this `EventType`. The predefined schema validator will use the value of
           this field as base for the validation.
-          The generation of this value happens on creation of the `EventType` and is influenced by the
-          `EventType.type`.
+
+          The createion of EventTypes of different categories implies a distinct general structure for the
+          payload of events of this each type. This ultimate format the payload must adhere to is reflected
```

Comment:
I don't know if I correctly understood the sentence "This ultimate format the payload must adhere to is reflected the `effective_schema`". Could you please rephrase it?